package.json

```diff
@@ -1,16 +1,10 @@
 {
-  "name": "styled-system",
-  "version": "4.2.2",
-  "description": "Responsive, theme-based style props for building design systems with React",
-  "main": "dist/index.cjs.js",
-  "module": "dist/index.esm.js",
-  "sideEffects": false,
+  "private": true,
   "scripts": {
-    "prepare": "mkdirp dist && npm run build:cjs && npm run build:esm",
-    "prettier": "prettier \"{src,test,examples}/**/*.js\" --write",
-    "clean": "rm -rf dist",
-    "build:cjs": "NODE_ENV=cjs babel src -o dist/index.cjs.js",
-    "build:esm": "NODE_ENV=esm babel src -o dist/index.esm.js",
+    "prettier": "prettier \"packages/**/{src,test,examples}/**/*.js\" --write",
+    "clean": "rm -rf packages/styled-system/dist",
+    "build": "lerna bootstrap",
```

Comment:
This is fine as-is, but I also don't mind using `npx lerna` in cases like this

Reply:
Noted! 👍